tools/snp.sh

```diff
@@ -413,6 +450,24 @@ set_rhel_grub_default_snp() {
   [ -d /sys/firmware/efi ] && sudo grub2-mkconfig -o /boot/efi/EFI/redhat/grub.cfg || sudo grub2-mkconfig -o /boot/grub2/grub.cfg
 }
 
+set_fedora_grub_default_snp(){
+  # Get the SNP host latest version from snp host kernel config
+  local snp_host_kernel_version=$(get_host_kernel_version)
+
+  # Retrieve snp menuitem name from the boot loader entries
+  local snp_menuitem_name=$(sudo sh -c 'grep title /boot/loader/entries/* | cut -d " " -f2-' \
```

Comment:
Split pipes on new lines. Indent.

Reply:
Yes, I did the above mentioned change. Please review